qiskit/ignis/characterization/calibrations/pulse_schedules.py

```diff
@@ -39,74 +39,53 @@ def rabi_schedules(amp_list, qubits, pulse_width, pulse_sigma=None,
         width_sigma_ratio: set sigma to a certain ratio of the width (use if
         pulse_sigma is None)
         drives: list of DriveChannel objects
-        cmd_def: CmdDef object to use (deprecated)
         inst_map: InstructionScheduleMap object to use
         meas_map: meas_map to use
 
     Returns:
        A list of QuantumSchedules
        xdata: a list of amps
-    """
 
+    Raises:
+        QiskitError: when necessary variables are not supplied.
+    """
     xdata = amp_list
 
     # copy the instruction to schedule mapping
     inst_map = copy.deepcopy(inst_map)
-    if not inst_map:
-        inst_map = copy.deepcopy(cmd_def)
+
+    # Following variables should not be optional.
+    # To keep function interface constant, errors are inserted here.
+    # TODO: redesign this function in next release
```

Comment:
I'd recommend opening an issue for this if you haven't already:)

Reply:
see #376